README.md

```diff
@@ -1,6 +1,6 @@
 # Gitcoin Governor Bravo
 
-An upgrade to a "Bravo" compatible Governor for the GitcoinDAO, built using the OpenZeppelin implementation.
+An upgrade to a "Bravo" compatible Governor for the GitcoinDAO, built using the OpenZeppelin implementation and [flexible-voting](https://github.com/ScopeLift/flexible-voting).
```

Comment:
Nit: can we just say [Flexible Voting](https://github.com/ScopeLift/flexible-voting) here?

Reply:
Sure, done in https://github.com/gitcoinco/Alpha-Governor-Upgrade/pull/20/commits/01372fbb1e1725747d7d7bb03bacc0f15d6ec3da